.gitignore

```diff
@@ -122,33 +122,36 @@ Session.vim
 tags
 
 
+#VSCode
+.vscode
+
 ### macOS ###
-*.DS_Store

```

Comment:
What's happening here?

Reply:
No idea, I don't use a Mac and don't remember touching that. I'll add it back

Reply:
I mean the entire chunk here - it looks like you changed line endings or something.